### PR TITLE
Add mantle chains

### PR DESCRIPTION
**Pull request by chengwenxi**
Mantle is a suite of Ethereum scaling solutions.

The Testnet and Mainnet will be launched in the next few months.

### CODE REVIEW

**Review by guoshijiang — APPROVED** [review]

**Review by shidaxi — APPROVED** [review]
LGTM

**Review by bradyjoestar — APPROVED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
eip155-5000json

there is a dot missing here

**Comment by chengwenxi** [comment]
> eip155-5000json
> 
> there is a dot missing here

Thanks for the review, added the missing dot.

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `MANTLE` more than once

**Review by ligi — CHANGES_REQUESTED** [review]
The explorers are not reachable

**Comment by chengwenxi** [comment]
> The explorers are not reachable

Once the blockchain is launched, explorers will be reachable.

**Comment by ligi** [comment]
ping me then - cant merge now as I cannot verify the explorers

**Comment by ligi** [comment]
no it's not - explorers are not reachable

**Comment by shidaxi** [comment]
@ligi our testnet sites are avaiable now but mainnet is WIP at the moment and not publicly accessible. We can roll out a initial version first. Now it's available. Pls check again https://explorer.mantle.xyz . Thank you very much!

**Comment by shidaxi** [comment]
Hey @ligi , is there any update?

File: _data/chains/eip155-5001.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://mantle.xyz",
+  "shortName": "mantle-testnet",
+  "chainId": 5001,
+  "networkId": 5001,
+  "explorers": [
+    {
+      "name": "Mantle Testnet Explorer",
+      "url": "https://explorer.testnet.mantle.xyz",
```
Comment (chengwenxi) [inline _data/chains/eip155-5001.json:18]:
```suggestion
      "url": "https://explorer.mantle.xyz",
```

Seems that the testnet explorer is same as mainnet, can you update it?

**Review by ligi — APPROVED** [review]

**Pull request merged**